sql-select/cheap-to-rent.sql

```diff
@@ -0,0 +1,4 @@
+     select "filmId", "title"
+       from "films"
+   order by "rentalRate"
```

Comment:
While this works in this situation it won't always work, it is best to use a where clause here to ensure we get no rentals with a `rentalRate` that is greater than a dollar.